javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,70 @@
+
+const LTag=document.getElementById("plL")
+const STag=document.getElementById("plS")
+const btn=document.getElementById("btn");
+const amountOftime=document.getElementById("amountOftime")
+const winner=document.getElementById("winner")
+const el = document.querySelector("p")
+const reset=document.getElementById("reset")
+let flag=true;
```

Comment:
`flag` is a bad name, you should call things what they are, for example `gameInProgress` or something like that, because it makes it easier to read the code when the value is used